packages/amplication-client/src/Resource/git/SyncWithGithubPage.tsx

```diff
@@ -94,7 +94,7 @@ const SyncWithGithubPage: React.FC = () => {
   const handleCommit = useCallback(() => {
     commitChanges({
       message: "",
-      projectId: currentProject.id,
+      project: { connect: { id: currentProject?.id } },
```

Comment:
In all other places in the client, we pass a `projectId` to `commitChanges` so I'm not sure why only here it's different - and if indeed this is the right way to pass this in, then it's better to make this "transformation" inside the hook and not in the caller